### PR TITLE
Update docker-compose stable to 1.5.0

### PR DESCRIPTION
**Pull request by kudos**
1.5.0 went stable yesterday. Stable and HEAD are equivalent in dependencies right now.

### CODE REVIEW

File: Library/Formula/docker-compose.rb [hunk 1]
```diff
@@ -62,6 +27,31 @@ class DockerCompose < Formula
...
+    sha256 "36673ac378feed3daa5956276a829699056523d7961027911f064b52255ead41"
+  end
+
+  resource "functools32" do
+    url "https://pypi.python.org/packages/source/f/functools32/functools32-3.2.3-2.tar.gz"
+    sha256 "f6253dfbe0538ad2e387bd8fdfd9293c925d63553f5813c4e587745416501e6d"
+  end
+
   resource "pyyaml" do
     url "https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz"
```
Comment (robsonpeixoto) [inline Library/Formula/docker-compose.rb]: Wrong PyYAML version. Must be 3.10
https://github.com/docker/compose/blob/master/requirements.txt#L1

**Comment by robsonpeixoto** [comment]
The six version should be `1.7.3`.
https://github.com/docker/compose/blob/master/requirements.txt#L8

**Comment by kudos** [comment]
They should all match now. For some reason the resources previously in `head do` were newer than the requirements file called for. I guess whoever originally added them just used the latest of each?

**Comment by robsonpeixoto** [comment]
To be honest, I have no idea. But I guess the same, @kudos.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/c32f04bccfe4ba95a088010cf4174adda3e5dc5c. Thank you for your contribution to Homebrew @kudos; we appreciate it! :smiley_cat:

**Pull request closed without merging**